compose/compose-file/compose-file-v2.md

```diff
@@ -902,7 +902,17 @@ then read-write will be used.
 >     - container_name
 >     - container_name:rw
 
-### cpu\_shares, cpu\_quota, cpuset, domainname, hostname, ipc, mac\_address, mem\_limit, memswap\_limit, mem\_swappiness, mem\_reservation, oom_score_adj, privileged, read\_only, restart, shm\_size, stdin\_open, tty, user, working\_dir
```

Comment:
I don't think we want to remove this heading. And we should probably change this in V3's reference too.